java-plugin/src/main/resources/fr/greencodeinitiative/l10n/java/rules/java/EC80.json

```diff
@@ -0,0 +1,16 @@
+{
+    "title": "Force the use of FetchType LAZY on collections in Entity JPA",
```

Comment:
now, it isn't the good way to declare rule properties but inside `ecocode-rules-specifications` (please see another rule as example)